### PR TITLE
add timestamps to conda debug logs

### PR DESCRIPTION
**Pull request by savingoyal**
to get output of the nature of
Screen Shot 2024-06-17 at 7 40 16 PM

### CODE REVIEW

**Comment by romain-intel** [comment]
Test failures are OK (unrelated dependency change)

File: metaflow/plugins/pypi/conda_environment.py [hunk 1]
```diff
@@ -178,11 +178,9 @@ def _path(url, local_path):
                 )
             if self.datastore_type not in ["local"]:
                 # Cache packages only when a remote datastore is in play.
-                storage = self.datastore(
-                    _datastore_packageroot(self.datastore, self.echo)
-                )
+                storage = self.datastore(_datastore_packageroot(self.datastore, echo))
```
Comment (romain-intel) [inline metaflow/plugins/pypi/conda_environment.py:181]: should this be self.logger?

**Review by romain-intel — APPROVED** [review]
Minor nit (we don't test this code so can't say conclusively) but otherwise can merge anytime.

File: metaflow/plugins/pypi/conda_environment.py (hunk 1 above)
Comment (savingoyal) [reply metaflow/plugins/pypi/conda_environment.py:181]: this one is intentional

**Pull request merged**